src/components/MatchBreakdownTable/index.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from "prop-types";
+import MatchBreakdown2019 from "./breakdown2019";
+
+const MatchBreakdownTable = ({ match }) => {
+  if (match.getYear() === 2019) {
+    return <MatchBreakdown2019 match={match} />;
+  }
+  return <h4>No Breakdown Found</h4>;
+};
+
+MatchBreakdownTable.propTypes = {
+  match: PropTypes.object.isRequired,
```

Comment:
Can you change this to `PropTypes.instanceOf(Match)` and `import Match from "../../database/Match"